db/academyPortal_2021-07-27.sql

```diff
@@ -0,0 +1,78 @@
+# ************************************************************
```

Comment:
please remove this file. there should be *one* database file, named the same as the original